libs/llm/llm.provider.service.ts

```diff
@@ -184,15 +186,14 @@ export class LLMProviderService implements OnModuleInit {
     const model =
       config.model || this.getDefaultChatProviderModel(config.provider);
 
-    const availableModels = this.getAvalableModels(config.provider);
+    const availableModels = this.getAvailableModels(config.provider);  // TODO - K: Maybe rename to "getAllowedModels"?
```

Comment:
These are just the models listed in the user config (.env). They are not "available". They might not even exist.

Reply:
Ok, available models comes from the respective APIs. However since most API provides all the models (e.g. embeddings and chat) the global env allow to filter out by name.